extensions/jackson/deployment/src/main/java/io/quarkus/jackson/deployment/JacksonProcessor.java

```diff
@@ -221,7 +220,8 @@ void register(
             AnnotationValue value = resolverInstance.value("value");
             if (value != null) {
                 // Add the type-id-resolver class
-                reflectiveClass.produce(new ReflectiveClassBuildItem(true, true, value.asClass().name().toString()));
+                reflectiveClass
+                        .produce(ReflectiveClassBuildItem.builder(value.asClass().name().toString()).methods(true).build());
```

Comment:
Shouldn't `fields()` be true too? Or it's not needed?

Reply:
Well spotted! Fixed